Makefile

```diff
@@ -13,7 +13,7 @@ check-format:
 
 .PHONY: check-style
 check-style:
-	flake8
+	flake8 mitiq
```

Comment:
Is this restriction necessary? E.g. what about `docs` (not sure if flake works with myst files)  and `benchmark` folders?

Reply:
Thanks for asking about this. It got me to dig a little deeper to understand the problem I was having previously. I was never able to run `flake8` on my machine. Rather the command would hang indefinitely. After some debugging I've found that I had both `.venv/` and `env/` directories it was going through checking style as well (but I no longer use virtualenv). So it was checking something like 20,000 files instead of 86. I've now removed those directories and it works fine. I'll revert this change!